### PR TITLE
Add CLX

### PR DESCRIPTION
**Pull request by dliul**
add a comment to Chuliuxiang

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: Chinese/CLX/clx.md [hunk 1]
```diff
@@ -0,0 +1 @@
+楚留香新传，应该是近几年来我看的最好的武侠片了。比起那些不要脸的做作，这部局可谓上上之作。
```
Comment (casey-collab) [inline Chinese/CLX/clx.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**